Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN pip install poetry
 RUN poetry config virtualenvs.create false
 COPY pyproject.toml poetry.lock ./
 RUN poetry install
+RUN pip install django-cors-headers
```

Comment:
You should install `django-cors-headers` via `poetry add` and remove this from the Dockerfile

Reply:
Yeah, I had some problems with that because certain versions of things didn't match my local versions so I did this for now. I'll fix before requesting a review.

Reply:
Ok cool, sounds good. It sounds like you might need to be inside `poetry shell`